assets/src/components/parts/janus-Carousel/Carousel.tsx

```diff
@@ -115,7 +116,8 @@ const Carousel: React.FC<PartComponentProps<CarouselModel>> = (props) => {
             break;
           case NotificationType.CONTEXT_CHANGED:
             {
-              const { initStateFacts: changes } = payload;
+              const { initStateFacts: initStateFacts } = payload;
+              const changes = getFormattedVariables(initStateFacts);
```

Comment:
Now, we are sending the init Variables as 'q:14541d:5421:1|stage.foo.val:20', we need to format that to 'stage.foo.val:20'